Automattic/jetpack
# Jetpack AI: Check for post id type and only include numbers

Fixes #

## Proposed changes:

* Checks if the post id provided is indeed a number, otherwise dropping it

`useSelect( select => select( 'core/editor' ).getCurrentPostId() );` returns a string like `pub/twentytwentythree//home` on the site editor (new behavior?), which caused all requests to fail on the site editor.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the site editor
* Use the AI Assistant block there
* Check that the requests work
